ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -71,7 +72,7 @@ cat >> "${CONFIG}" << EOF
 platform:
   openstack:
     cloud:             ${OS_CLOUD}
-    computeFlavor:     ${OPENSTACK_COMPUTE_FLAVOR}
+    computeFlavor:     ${OPENSTACK_CONTROLPLANE_FLAVOR}
```

Comment:
While you're at it, we deprecated `computeFlavor` in 4.7. We should now use the `type` attribute of the machine pools, like done below for the master machine pool. You should remove this line.

Reply:
Won't that break 4.6 jobs? I personally remember having issues when not setting this. I'll give it a try again against 4.6 and 4.7 to see how it behaves.

Reply:
no you're right this should work: https://github.com/openshift/installer/pull/4394